### DISCUSSION TITLE
ja3-fingerprints

### DISCUSSION THREAD

**Discussion by Jackson-Pollock** [Security-Onion-Solutions/securityonion#6562] [Unsupported Versions] [answered]
I want to utilize the JA3 integration of Suricata and I changed `app-layer.protocols.tls.ja3-fingerprints` to `yes` in `suricata/defaults.yaml` but that seems doesn't work. Require rules are already available in `all.rules`. 
Is there any additional config needed? Using it for the first time. 
Thanks in advance!

**Comment by Jackson-Pollock** [comment, marked as answer]
Any suggestions?

**Reply by Jackson-Pollock** [reply]
Below is the config and rules status
```
# cat /opt/so/saltstack/local/salt/suricata/defaults.yaml | grep -i Ja3
          ja3-fingerprints: auto

# cat /opt/so/log/suricata/suricata.log | grep -i signatures
15/12/2021 -- 04:34:02 - <Info> - 23869 signatures processed. 1247 are IP-only rules, 4020 are inspecting packet payload, 18577 inspect application layer, 0 are decoder event only
```

**Reply by Jackson-Pollock** [reply]
I tried importing some of pcap files (Ja3 detectable) but no Ja3 alerts found.

**Reply by tomjones1977** [reply]
Strange that it doesn't work as you expect, I had JA3 hash alerts pop up from the ET ruleset out of the box, with no extra configuration.

**Reply by Jackson-Pollock** [reply]
working.
